conanfile.txt

```diff
@@ -0,0 +1,3 @@
+[requires]
+catch2/3.7.0
```

Comment:
Is this the latest catch release on conan center?

Reply:
Ok it's at the latest 3.8.0 now